lib/addons/vpc-cni/index.ts

```diff
@@ -375,6 +375,7 @@ export class VpcCniAddOn extends CoreAddOn {
     (this.coreAddOnProps.configurationValues as any) = populateVpcCniConfigurationValues(props);
   }
 
+  @autoMode()
```

Comment:
does not seem like we should fail here, it is in the list of addons that must be above a particular version. So for vpc-cni minimum version should be 1.19.0-eksbuild.1. It seems like adjust the decorator to take the expected min version would be a better path, e.g. conflictsWithAutomode(version: version).